canonical/ubuntu.com
# refactor: Rename openstack chart partial so it doesn't appear as a page

## Done

- Previously the chart for openstack releases was called `openstack-release-cycle.html`, which mean it appeared as a standalone page under `/about/openstack-release-cycle`. This renames it to `_openstack-release-cycle.html`

## QA

- Check out https://ubuntu.com/about/openstack-release-cycle
- Now check out https://ubuntu-com-14277.demos.haus/about/openstack-release-cycle and see it no longer exists
- Also see https://ubuntu-com-14277.demos.haus/about/_openstack-release-cycle doesn't exist
- Finally, check https://ubuntu-com-14277.demos.haus/openstack/why-openstack and see the partial is included

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14795
